.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: java
+
+jdk:
+  - oraclejdk8
+
+notifications:
+  email: false
+  
+# whitelist
+branches:
+  only:
+    - trilok_travis_ci
+    - master
```

Comment:
Shouldn't this be on all branches for which PRs are created? That way one can see the status of precheckin before a merge (and a successful pass will become a requirement for merge) rather than after the merge.

Reply:
Sure. Just demoing that it is possible to whitelist or exclude branches. I can comment out or remove this part. 
The idea of pull request builds is to add a pre-checkin successful build verification. This build is triggered on creating a PR, not at the time of merge.